README.md

```diff
@@ -264,8 +264,9 @@ or on the command line:
 	-c, --cert-pem=filename       DTLS certificate
 	-k, --cert-key=filename       DTLS certificate key
 	-K, --cert-pwd=text           DTLS certificate key passphrase (if needed)
-	-S, --stun-server=url    STUN server(:port) to use, if needed (e.g.,
-								  Janus behind NAT, default=none)
+	-S, --stun-server=address:port
+                                  STUN server(:port) to use, if needed (e.g.,
```

Comment:
I've followed how the formatting was done for `-m, --reclaim-session-timeout=number` as that was another parameter that forced the start of the description onto the next line

Reply:
It should be enough to recompile and launch Janus with the `--help` property, and copy it in the README, and then just indent  the first line with a tab instead of spaces (for the other lines you can keep what is generated).

Reply:
I mean just copying the updated property, of course, not all of them.

Reply:
It seems it doesnt want to compile on mac for me... so I'll have to sort this out tomorrow.

Reply:
No need, I can fix it myself after merging :+1: